### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by derpachu** [derpachu/MI-449-SS17-740-js-conditionals-uMDt4P#1]
@chrisvfritz Can you take a look at this? It's [hosted here](https://derpachu.github.io/MI-449-SS17-740-js-conditionals-uMDt4P/) and meets the following criteria:

- [x] Use the `prompt` and `confirm` functions (each at least once) to collect choices from the user (the entire story will take place in dialog boxes, so your `body` element could actually be empty, apart from the `script` element to include your JavaScript)
- [x] Use `if`, `else if`, and `else` to branch the story, based on user choices
- [x] In at least one branch, there should be at least 3 choices for the user to make
- [x] Include at least one branch where different things can happen each time with `Math.random()`
- [x] Use a logical operator (`&&` or `||`) at least once, where it makes sense
- [x] It should be impossible for an error to ever occur when playing your game, including handling for `null`/`undefined` where appropriate
- [x] All string input should be normalized, so that extra whitespace or a letter that's the wrong case will never cause a failed match
- [x] At least one number value is collected and used in a comparison (don't forget to convert it into a number first)
- [x] Don't get _too_ carried away -- it can be tempting to build a complex video game, but remember, you can always come back and build on this project later

Wrote the javascript python style, is there any habits of writing javascript that I should know about?

**Comment by chrisvfritz** [comment]
Sorry it took so long to get a review on this. I really, really like this project!! Definitely one of my the nicest implementations I've seen so far. 🚀 💥 🎈🔥 🎊 😄  I especially like:

- Clearly thought-out gameplay, with great handling of user input. Possibly the only exception is that there's no way to quit currently, if a user decides they want to part-way through.
- Overall well-named variables and functions.
- Great use of functions and loops.

I don't want to hold you up on this project any longer than I already have with my late feedback, so I'm giving this a :shipit:, but since you asked about style in JavaScript, I think my biggest piece of feedback would be that more and smaller functions are never a bad thing. 🙂  

I also just put together [a little refactor](https://gist.github.com/chrisvfritz/55caf42839a4e2e0c060c38dbd108c25) (including some arrays, since you have prior programming experience), with comments to give you an idea of some areas for improvement. Let me know what you think and if you have any questions/comments! 😄

**Comment by derpachu** [comment]
Will things like dictionary eventually be covered or will be more array like?

**Comment by chrisvfritz** [comment]
Yes! There's currently a lesson about objects, which are like dictionaries.
